src/winstanley/Wdl.flex

```diff
@@ -27,7 +27,10 @@ BOOLEAN="true"|"false"
 NUMBER="0"|"1"|"2"|"3"|"4"|"5"|"6"|"7"|"8"|"9"
 
 WHITE_SPACE= \n|\r|\r\n|\ |\t|\f
-TYPE="Array"|"Map"|"Object"|"Boolean"|"Int"|"Float"|"Uri"|"File"|"String"
+PRIMITIVE="Object"|"Boolean"|"Int"|"Float"|"Uri"|"File"|"String"
+ARRAY="Array"
+PAIR="Pair"
+MAP_TYPE="Map"
```

Comment:
`MAP` was already used in the BNF.  I will add `_TYPE` to the others for consistency